src/ui/components/Library/Library.module.css

```diff
@@ -272,3 +274,45 @@
 .dropdownTrigger:focus {
   box-shadow: 0 0 0 var(--ring-width) var(--ring-color);
 }
+
+.testTitle {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  gap: 0.25rem; /* For 'gap-1' */
+  border-bottom: 1px solid var(--theme-border);
+  padding: 1.065em 1rem;
+}
+
+.testTitle > div {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  font-size: 1.125rem;
+  font-weight: 500;
+}
+
+.replayRow {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  gap: 0.5rem;
+  border-bottom: 1px solid var(--chrome);
+  padding: 0.625rem 1rem;
+  cursor: pointer;
+  transition: background-color 0.2s;
+}
+
+.replayListTitle {
+  padding: 0.5rem 1rem 0.08rem;
```

Comment:
The `0.08rem` is a very specific choice too. Do we do this elsewhere?

Reply:
Same as comments above. Here's how it looks:
<img width="135" alt="image" src="https://github.com/replayio/devtools/assets/9154902/8a09cd3b-4055-4be1-8fdf-5a149eaa22fe">

Here's how it would look if I stuck with round numbers:
<img width="58" alt="image" src="https://github.com/replayio/devtools/assets/9154902/52adf5b0-f3d2-407e-86fb-6068fcc142d2">

Reply:
I assumed you had a valid reason for the sizing but I do wonder if this approach addresses the symptom instead of the problem. Should we adjust the sizes or margins of the contents of these containers to be consistent instead?

Reply:
Fair question, I'm glad you asked!

So if you have two components that share most characteristics, like text inside a tab, then it's easy to keep things the same across height, padding, margin, etc. You still run into optical alignment issues like how descenders make the bottom spacing look off, but it's a fine place to start.

<img width="1302" alt="image" src="https://github.com/replayio/devtools/assets/9154902/02b53bc6-016b-49ff-9bb1-af8c4e0167d1">

The issue is that nearly nothing is this simple. For example, if you need to position a dropdown inside a header or a tab, you're dealing with all sorts of different math. In this example, I have maintained a perfect 8px alignment all the way around the element. But it looks like garbage, because the dropdown needs its own layout needs to be met. This is an example of "consistency" making a worse design.

<img width="1256" alt="image" src="https://github.com/replayio/devtools/assets/9154902/54a0cc88-fee0-4340-860e-1720b5a13e71">

Something will have to be made inconsistent. One idea is to reduce the size of the text, so that's inconsistent but at least our precious margins are consistent! Still looks like garbage.

<img width="1011" alt="image" src="https://github.com/replayio/devtools/assets/9154902/f3d226b6-7bef-4bcb-b4b8-9bd0502c3d3a">

So a better thing to do here is pop out the margins so the text can stay consistent. This means the margins are "inconsistent" but they're optically well-aligned.

<img width="1206" alt="image" src="https://github.com/replayio/devtools/assets/9154902/8b9bdc70-30e3-49cf-8052-3320496604f1">

--

So that's where we are with this. If you look at the dropdowns on the left, that region is 51px tall:
![image](https://github.com/replayio/devtools/assets/9154902/00b3d738-ecea-4f99-afae-09e8f2b7b3c8)

Why 51? Because there's 1px for the border, then 8px + 8px for the top and bottom margins, and 34 pixels tall for the drop-down themselves:

![image](https://github.com/replayio/devtools/assets/9154902/27851f49-c679-4a61-9284-80c627bb4b53)

Which means the item on the right needs to have a fixed font size (14px), aligned optically with the drop-down on the left, and we're trying to reach 51px total height. We also want it to be in REM rather than pixels so it scales correctly with the text. If you pick 1 REM, it's a bit too tall. So we season to taste.

--

In these examples, the key is consistency *in the final design that users see* rather than consistency in the code. Both is ideal, but when they clash (as they do when you're mixing components of different sizes) we need to optimise for the user, not the machine.

Whew! Design class!

Reply:
> I assumed you had a valid reason for the sizing but I do wonder if this approach addresses the symptom instead of the problem.

This has been my feeling too when seeing odd units like this.

Reply:
I'd like to resolve this once and for all because frankly it takes a lot of time to explain and re-explain.

It's true that sometimes a strange number points to a larger problem. In our codebase, we've usually swung the other way, where our design looks sloppy because lines just barely don't line up. As said above, we need to optimise for what the user sees, which sometimes requires fine-tuning like this.

Reply:
I wasn't asking you to re-explain FWIW. We just disagree here.

Reply:
I'd appreciate fewer comments about my "bizarre" layouts and my "arbitrary" numbers. The fact is our app looks broken when basic layouts don't line up. You could even say it looks "bizarre" to users.

Reply:
I leave these comments because in my experience of maintaining frontend apps, atypical values are difficult to maintain and the complex relationships between various padding/margin/styles that require those values is often also easy to break.

Reply:
No worries at all Jon. We all say things sometimes in the heat of the moment. I appreciate your follow up message!